card/src/main/java/com/adyen/checkout/card/CardView.kt

```diff
@@ -408,6 +412,11 @@ internal class CardView @JvmOverloads constructor(
         binding.cardBrandLogoContainerSecondary.setOnClickListener(null)
     }
 
+    private fun deselectBrands() {
```

Comment:
naming could be better, will fix it. leave a comment for your suggestions if you have any :v:

Reply:
I think it should be something like `initializeUnselectedCardBrandLogos()` since it is gonna be valid for initial state. When you read this you might think it's possible to deselect after selecting one.